src/Files.Uwp/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml

```diff
@@ -592,10 +591,23 @@
                         Style="{StaticResource Local.TabViewItem}"
                         ToolTipService.ToolTip="{x:Bind HoverDisplayText, Mode=OneWay}" />
                 </DataTemplate>
-            </muxc:TabView.TabItemTemplate>
-            <muxc:TabView.TabStripFooter>
-                <Border HorizontalAlignment="Left" Child="{x:Bind ActionsControl, Mode=OneWay}" />
-            </muxc:TabView.TabStripFooter>
-        </muxc:TabView>
+            </TabView.TabItemTemplate>
+            <TabView.TabStripFooter>
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+                    <ContentPresenter Content="{x:Bind ActionsControl, Mode=OneWay}" />
+
+                    <!--  This defines the drag area for the title bar  -->
+                    <Grid
```

Comment:
I think we need to keep the drag area on the home page, otherwise it overlaps with the tabs.

Reply:
What do you mean "it overlaps with the tabs"? I moved it out of main page because otherwise the tabs and "+" button were not clickable. Putting the drag area element behind (z-order) other elements may not work:
> The rectangular area occupied by the specified element acts as the title bar for pointer purposes, even if the element is blocked by another element, or the element is transparent.

Reply:
UWP had a weird setup with this and I'm not sure how much changed but I'll take a look. As it is right now, tabs aren't clickable (there is a small section at the bottom of the title bar that works.

Reply:
Funny I can click on the tabs just fine.

Reply:
I'll check again but perhaps Win10 is different